src/css/objects/_media.scss

```diff
@@ -245,172 +241,216 @@
+    padding-bottom: 0.5rem;
+  }
 }
 
-.media--hp {
-  box-shadow: inset 17rem 0 0 $color-gray-lightest;
-  padding: $base-line-height 0 $base-line-height 19rem;
+.o-media--med {
+  // box-shadow: inset 17rem 0 0 $color-gray-lightest;
+  // padding: $base-line-height 0 $base-line-height 19rem;
   position: relative;
 
+  //NOTE: Re-add the following padding as part of the CSS specific to the homepage rather than here.
+  // padding-top: 4.25rem
+
+  .o-media__figure {
+    margin-bottom: 1.375rem;    
+  }
+
   @include media-max($media-tablet) {
     border-top: 1px solid $color-gray-light;
     box-shadow: none;
     padding-left: 0;
-  }
 
-  &::after {
-    border-top: 1px solid $color-gray-light;
-    content: '';
-    display: block;
-    left: 17rem;
-    position: absolute;
-    right: 0;
-    top: 0;
+    .o-media__content {
+      // If flex is available, and we are on mobile,
+      // use this to position the figure before the 
+      // rest of the media content.
+      display: flex;
+      flex-direction: column;
+      .o-media__figure {
+        order: -1
+      }
+    }
 
-    @include media-max($media-tablet) {
-      display: none;
+    .o-media__img {
+      margin: 0 auto;
     }
-  }
 
-  > .media__figure {
-    margin: 0 0 $base-line-height;
   }
 
-  > .media__headline {
-    margin: 0 0 $base-line-height;
-  }
+  @include media-min($media-tablet){
+    $width: 18.625rem;
 
-  > .media__label {
-    font-size: ms(-2);
-  }
+    [class*='o-media__figure--'] {
+      .o-media__img {
+        box-sizing: border-box;
+        left: 50%;
+        position: absolute;
+        top: 50%;
+        transform: translate(-50%, -50%);
+      }
+    }
 
-  .media__overline {
-    color: $color-secondary;
-  }
+    .o-media__figure {
+      margin-right: 3.125rem;
+      float: left;
+      max-width: $width;
+    }
 
-  .media__related-content {
-    margin-left: -2rem;
-    padding-left: 2rem;
-    padding-right: 2rem;
+    .o-media__figure--cinematic {
+      height: $width / 2.35;
+    }
 
-    @include media-max($media-tablet) {
-      margin-left: 0;
-      padding-left: 0;
-      padding-right: 0;
+    .o-media__figure--four-by-three {
+      height: $width / 1.33333;
     }
 
+    .o-media__figure--widescreen {
+      height: $width / 1.77778;
+    }
   }
 
 }
 
-.media--hp-large {
+.o-media--lg {
   margin-top: $base-line-height;
 
-  > .media__content {
-    box-shadow: inset 17rem 0 0 $color-gray-lightest;
-    padding: $base-line-height 0 ($base-line-height * 2) 19rem;
+  > .o-media__content {
+    // box-shadow: inset 17rem 0 0 $color-gray-lightest;
+    // padding: $base-line-height 0 ($base-line-height * 2) 19rem;
 
     @include media-max($media-tablet) {
       box-shadow: none;
       padding-left: 0;
     }
-
   }
 
-  > .media__figure {
+  .o-media__figure {
     border-bottom: 6px solid $color-primary-alt;
-    margin: 0 ($gutter * -1);
+
+    // margin: 0 ($gutter * -1) //NOTE: Unclear why we need this.
```

Comment:
We were using this in the old homepage design to pull large assets out beyond the boundary of its container - it was a design detail that was part of the old implementation. We no longer need this on large screens - it can be removed.